google/cloud/logging_v2/logger.py

```diff
@@ -457,6 +461,36 @@ def commit(self, *, client=None):
             kwargs["labels"] = self.logger.labels
 
         entries = [entry.to_api_repr() for entry in self.entries]
-
-        client.logging_api.write_entries(entries, **kwargs)
+        try:
+            client.logging_api.write_entries(entries, partial_success=True, **kwargs)
```

Comment:
As I mentioned before, seems that in https://cloud.google.com/logging/docs/reference/v2/rest/v2/entries/write the `partialSuccess` is optional

Reply:
The partialSuccess field is being [modified by a different PR](https://github.com/googleapis/python-logging/pull/649/files#), so you can ignore that here

Although this improved exception message feature will only work if `partialSuccess` is `True`, which might be another reason to keep it on by default